tests/operator/operator.go

```diff
@@ -888,11 +893,11 @@ spec:
 			Eventually(fetchVirtHandlerCommand, 60*time.Second, 5*time.Second).Should(ContainSubstring(maxDevicesCommandArgument))
 
 			By("Deleting patch from KubeVirt object")
-			kv, err = virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Get(context.Background(), originalKv.Name, metav1.GetOptions{})
+			newVMIPerNodeRemovePatch, err := patch.New(
+				patch.WithRemove("/spec/configuration/virtualMachineInstancesPerNode")).GeneratePayload()
 			Expect(err).ToNot(HaveOccurred())
 
-			kv.Spec.Configuration.VirtualMachineInstancesPerNode = nil
-			kv, err = virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Update(context.Background(), kv, metav1.UpdateOptions{})
+			kv, err = virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Patch(context.Background(), kv.Name, types.JSONPatchType, newVMIPerNodeRemovePatch, metav1.PatchOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Waiting for virt-operator to apply changes to component")
```

Comment:
I think that we can make this test better by verifying that the all virtualization nodes have `/status/allocatable: devices.kubevirt.io/kvm=10`
Is this something you would consider @orenc1 ?

Reply:
yes, this can be a good indicator that the `virtualMachineInstancesPerNode` config has made an actual impact on the nodes' allocatable kvm resources.

Reply:
a test case has been added that checks that all KVM-capable nodes expose the desired allocatable kvm devices amount, as per `virtualMachineInstancesPerNode`.